sdk/eventgrid/Azure.Messaging.EventGrid/tests/EventGridClientLiveTests.cs

```diff
@@ -45,7 +45,6 @@ public async Task CanPublishEventToDomain()
                     new AzureKeyCredential(TestEnvironment.DomainKey),
                     options));
             await client.PublishEventsAsync(GetEventsWithTopicsList());
-
         }
 
         [Test]
```

Comment:
Can we add tests for CloudEvents/EventGridEvents where a custom object is used for payload? Also one where the Data is null for CloudEvent?

Reply:
Just the null data payload test remaining.

Reply:
Oh the original CanPublishCloudEvent test actually has null data payloads